docs/src/Accessibility.doc.js

```diff
@@ -0,0 +1,195 @@
+      `}
+    />
+    <MainSection.Subsection
+      title="Interaction"
+      description={`
+    Accounting for all aspects of interaction is critical to creating inclusive designs.
+
+    - **Create designs for every state**, like focus, hover, focus + hover, active, selected, disabled, and anything else that might be relevant. Ensure each combination is visually distinct from the others.
+    - **Keep disabled elements disabled.** Disabled buttons are removed from the tab order, so they are impossible to access with a keyboard. Adding something like a tooltip to a disabled item means only mouse-users can access the content of the tooltip, so the best option is to attach the tooltip to an active element, like an IconButton, next to the disabled element instead.
+    `}
+    />
+    <MainSection.Subsection
+      title="Comprehension"
+      description={`
+    Seemingly small design decisions can have a large impact on users comprehension.
+
+    - **Use form labels whenever possible.** Placeholder text is not enough to indicate to users what information they are expected to enter in any given text field, since it disappears as soon as any content is typed. Adding labels to every form field and dropdown will make expectations clearer for everyone.
+    - **Design a logical heading hierarchy.** Screen reader users can quickly navigate an entire page by analyzing the headings provided. Be sure to follow a logical order, starting with a single H1, and progressing to H2, H3, etc. If you could see only your headings, would the page make sense?
+    - **Create descriptive link text.** Avoid multiple links with vague text like "More info" or "Learn more". Instead, users should be able to tell exactly what info will be provided at a given link by the link text only. For instance, "Explore campaigns" instead of just "Learn more".
+    `}
+    />
+    <MainSection.Subsection
+      title="Responsiveness"
+      description={`
+    Accessibility requirements vary depending on the device used, so be sure to account for small and large screens alike.
+
+    - **Account for zooming!** Many users, especially on mobile, adjust the default size of fonts and often increase the zoom setting. Does your design still work when zoomed in 200%?
+    - **Create large touch targets.** If you've ever clicked on the wrong thing because the tap target was too small, you know how frustrating it can be to have to re-orient yourself. Ensure your touch targets on mobile are large enough for people with limited mobility to accurately press what they're aiming for.`}
```

Comment:
> Ensure your touch targets on mobile
I don't think this is only on mobile, right? I see tap targets that are way to small on desktop as well

Reply:
definitely applies to both, but is much more prevalent on mobile. I'll update